src/components/settings-row/settings-row.stories.mdx

```diff
@@ -44,6 +44,19 @@ All `children` are rendered in the input column to the right of the header colum
   </Story>
 </Canvas>
 
+### with HeadingType
+
+The `headingType` prop changes the HTML heading element of the `title` string which is displayed within the `SettingsRow` component. Each `string` 
+represents a HTML Section Heading element e.g `h1` is equal to `<h1>`, allowing up to six sections of headings from most to least important.
```

Comment:
same comment as for the new story in `Heading` (looks like you copy-pasted this, which is fair enough - just ensure that if you change it the new version is consistent across both components!)